packages/shell-chrome/views/settings/index.edge

```diff
@@ -0,0 +1,68 @@
+<div
+    data-testid="settings-panel"
+    x-show="settingsPanelOpen"
+    class="inset-0 absolute flex flex-col items-center justify-center z-max"
+>
+    <div class="absolute bg-alpine-400 opacity-30 inset-0"></div>
+    <div class="p-4 flex-1 flex items-center justify-center w-full relative z-50" @click="settingsPanelOpen = false">
+        <div
+            x-show.transition.in.opacity="settingsPanelOpen"
+            class="bg-white w-full h-full p-4 shadow-lg border border-silver-400 relative"
+            @click.stop
+        >
+            <button
+                data-testid="settings-panel-close-button"
+                class="absolute top-0 right-0 focus:outline-none block"
+                @click="settingsPanelOpen = false"
+            >
+                <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 20 20"
+                    class="text-black absolute top-1 right-1 w-5 h-5"
+                    fill="currentColor"
+                >
+                    <path
+                        fill-rule="evenodd"
+                        d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
+                        clip-rule="evenodd"
+                    />
+                </svg>
+            </button>
+            <div
+                x-data="{
+
+                    pages: [
+                        {
+                            name: 'Preferences',
+                            enabled: true,
+                        },
+                        {
+                            name: 'Experimental',
+                            enabled: window.devtools().experimentalPanelEnabled,
```

Comment:
could we toggle this using an Edge.js interpolation instead?

Would mean we can remove `experimentalPanelEnabled` from devtools function

Reply:
Can you explain/expand on that a bit more? I'm not sure what you mean exactly.

Reply:
https://edge.adonisjs.com/docs/getting-started#_rendering_plain_string you can pass the feature toggle in `edge.render()` which is in https://github.com/alpine-collective/alpinejs-devtools/blob/master/lib/edge/render.js#L10 and then interpolate it out using `{{ }}` here

Reply:
Ah ok yeah that looks right. I'll mess around with that tomorrow